deps/obs-scripting/obs-scripting-lua-frontend.c

```diff
@@ -297,37 +297,45 @@ static int add_save_callback(lua_State *script)
 
 /* ----------------------------------- */
 
+typedef struct lua_function_tuple {
+	const char *name;
+	lua_CFunction function;
+} obs_frontend_func;
+
+static obs_frontend_func functions[19] = {
```

Comment:
```suggestion
static const obs_frontend_func functions[] = {
```

Reply:
Ah, I remember again why I did it that way - Cl.exe doesn't seem to like that:

```
D:\a\obs-studio\obs-studio\obs-studio\deps\obs-scripting\obs-scripting-lua-frontend.c(305,46): error C2099: initializer is not a constant [D:\a\obs-studio\obs-studio\obs-studio\build64\deps\obs-scripting\obs-scripting.vcxproj]
```

Reply:
It's indeed the cast - `clang` and `gcc` don't mind, but `cl` doesn't like it. Easy enough to fix.